src/content/observer.ts

```diff
@@ -58,10 +68,12 @@ export const TUICObserver = {
 
         TUICObserver.functions.fixDMBox();
         if (window.location.pathname == "/tuic/safemode") {
-        } else if (document.querySelector("#unsent-tweet-background") == null && document.querySelector('[role="slider"]:not(article *)') != null && window.location.pathname == "/settings/display") {
-            TUICOptionHTML.displaySetting(document.querySelector('[role="slider"]:not(article *)').parentElement.parentElement.parentElement.parentElement.parentElement.parentElement.parentElement);
-        } else if (document.querySelector("#unsent-tweet-background") == null && document.querySelector('[role="slider"]:not(article *)') != null && window.location.pathname == "/i/display") {
-            TUICOptionHTML.displaySetting(document.querySelector('[role="slider"]:not(article *)').parentElement.parentElement.parentElement.parentElement.parentElement.parentElement.parentElement.parentElement);
+        } else if (window.location.pathname == "/settings/display") {
+            TUICLibrary.waitForElement(`section[aria-labelledby="detail-header"] > div.r-qocrb3`).then(() => {
+                TUICOptionHTML.displaySetting(document.querySelector('section[aria-labelledby="detail-header"] > div.r-qocrb3'));
+            });
+        } else if (window.location.pathname == "/i/display") {
+            TUICOptionHTML.displaySetting(document.querySelector(`[aria-labelledby="modal-header"] > div > div > div`).children[1]);
```

Comment:
```suggestion
            TUICLibrary.waitForElement(`[aria-labelledby="modal-header"] > div > div > div > div:nth-child(2)`).then(() => {
                TUICOptionHTML.displaySetting(document.querySelector(`[aria-labelledby="modal-header"] > div > div > div > div:nth-child(2)`));
            });
```
こうすれば最初は表示される(ついでに`.children[1]`じゃなくて`:nth-child(2)`を使う)